net-libs/cppzmq/cppzmq-4.6.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="High-level CPP Binding for ZeroMQ"
+HOMEPAGE="https://github.com/zeromq/cppzmq"
+SRC_URI="https://github.com/zeromq/${PN}/archive/v${PV}.tar.gz ->  ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+
+# Except for test these are used to alter installed cmake files:
+IUSE="test drafts"
+REQUIRED_USE="drafts? ( test )"
```

Comment:
Why would building draft API require test dependencies?

Reply:
Because the flag only influences the test process. I can drop it altogether, but it is not clear how to do the tesing of draft API in this case.

Reply:
Maybe use `has_version` to determine whether zmq was built with draft flag.

Reply:
draft api is optional even if zeromq is build with support of it

Reply:
Does that prevent you from testing it?

Reply:
No, but I don't think that significant changes in draft api should always lead to test errors when zeromq is build with corresponding use flag. That's just IMHO though. I can force has_version check if you say it's the right way to go. 
Perhaps it is possible to identify all the tests that use draft api and mark them as `known failures` somehow.

Reply:
Always test as much as you reasonably can. We can revisit this if things start failing.

Reply:
ok, done